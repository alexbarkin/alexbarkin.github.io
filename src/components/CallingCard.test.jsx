import React from "react"
import renderer from "react-test-renderer"
import { shallow } from "enzyme"

import CallingCard from "./CallingCard.jsx"

describe("Calling Card", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<CallingCard />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders with an image", () => {
    const sut = shallow(<CallingCard />)
    expect(sut.find("img").exists()).toBe(true)
  })
})
