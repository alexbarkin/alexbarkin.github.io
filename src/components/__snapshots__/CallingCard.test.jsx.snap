// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container--address"
>
  <address
    className="container--address-me"
  >
    <img
      alt="headshot"
      className="headshot"
      src="test-file-stub"
    />
    <p
      className="container--address-text"
    >
      Alex Barkin
      <br />
      Linkedin: 
      <a
        href="https://www.linkedin.com/in/abarkin/"
      >
        abarkin
      </a>
      <br />
      Github: 
      <a
        href="https://github.com/alexbarkin"
      >
        alexbarkin
      </a>
      <br />
      Email: 
      <a
        href="mailto:alex.barkin@edu.uwaterloo.ca"
      >
        alex.barkin@edu.uwaterloo.ca
      </a>
      <br />
    </p>
  </address>
  <iframe
    allowtransparency="true"
    className="instagram"
    frameBorder="0"
    scrolling="no"
    src="https://snapwidget.com/embed/751863"
    title="instagram"
  />
</div>
`;
